Tests/SourceKitLSPTests/ImplementationTests.swift

```diff
@@ -36,7 +37,8 @@ final class ImplementationTests: XCTestCase {
       ws.testLoc(name)
     }  
     func loc(_ name: String) -> Location {
-      Location(badUTF16: ws.testLoc(name))
```

Comment:
Shouldn’t `ws.testLoc` already return a native URI?

Reply:
It currently does not.  I can certainly try to see if we can get away with it, but that would change a number of location spellings, and I am worried that it might result in new failures.

Reply:
Oh, OK. I just thought it might be easier if we change the return value of `testLoc`. I’m fine to keep it as-is for now.